receiver/awscontainerinsightreceiver/config.go

```diff
@@ -60,4 +60,8 @@ type Config struct {
 
 	// EnableGpuMetric toggles GPU monitoring where metrics are scraped from vendor specific sources
 	EnableGpuMetric bool `mapstructure:"gpu_metrics"`
+
+	// EnableNeuronMetric disables Neuron monitoring where metrics are scraped from vendor specific sources
+	// The default value is true meaning Neuron metrics get collected out of the box unless it's disabled
+	EnableNeuronMetric bool `mapstructure:"neuron_metrics"`
```

Comment:
Are you missing adding a line like [this one](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/852e62a5c9477966dd9f21221b87c48601a4da67/receiver/awscontainerinsightreceiver/factory.go#L66) in the receiver default config?

Also, I think the default value should be false to match the control plane metrics.  In CWA we will default it to true if the user has turned on Enhanced Container Insights.